doc/python.md

```diff
@@ -0,0 +1,237 @@
+# Python Support
+
+Spark Job Server supports Python jobs through a Python specific context factory 
+`spark.jobserver.python.PythonSparkContextFactory`. See the [Contexts](contexts.md) documentation
+for information on contexts.
+    
+## Setting up Spark Job Server with Python support
+
+The `PythonSparkContextFactory` class is part of `job-server-extras`, therefore it is the assembly jar of that sub-module
+which should be used (this is the default for deployment anyway).
+
+The application config can be configured with paths to be added to the `PYTHONPATH` environment variable when the python
+subprocess which runs the jobs is executed. At a minimum, it should include the path of the Job Server API Egg file and
+the location of the Spark python libraries (which are most conveniently referenced in relation to `SPARK_HOME`. 
+ 
+By default python jobs are executed by calling an executable named `python`. In some cases, an alternative executable
+may be desired, for example to use Python 3 or to use an executable with an absolute path rather than assuming the 
+executable can be located on the `PATH`.
+ 
+Python binaries are only supported in the SQL flavour of `JobDAO` therefore it is also necessary to specify this in the config.
+  
+A basic config supporting Python might look like:
+
+    spark {
+      jobserver {
+         jobdao = spark.jobserver.io.JobSqlDAO
+      }
+    
+      context-settings {
+        python {
+          paths = [
+            ${SPARK_HOME}/python,
+            "/home/kss_admin/code/sjs/spark-jobserver/job-server-extras/job-server-python/target/python/spark_jobserver_python-0.7.0_SNAPSHOT-py2.7.egg"
+          ]
+          
+          #This can be omitted to use the default value of "python"
+          executable = "python3"
+        }
+      }
+    }
+
+There are two python packages which need to be provided, `py4j` and `pyhocon`. These can either be installed into the
+python instance to be used for running the jobs, or added to the list of paths in the config.
+
+In development, you can run a Python ready job server locally with the following `reStart` task:
+
+    job-server-extras/reStart ../job-server-python/config/python-example.conf.template
```

Comment:
Hmmm... not a biggie but I'd rather consolidate the Python configs into the same config/ dir as everything else